### PR TITLE
contact us form, new logo & design system

### LINKED ISSUE #36: Contact page v2
Jannik designed a new contact page. We can implement this in the future. Low priority because we already have a decent contact page.

image (1)

### LINKED ISSUE #51: update icon

### PR DESCRIPTION
**Pull request by JannikWibker**
closes #51 
closes #36

### CODE REVIEW

**Comment by JannikWibker** [comment]
For now this draft PR is just here in order to not forget about the contact us form design
Functionality (submitting form data) will follow soon™️

**Comment by use-to** [comment]
This pull-request scope is way too large. We need faster and more iterations. So others don't wait for this...
Try to finish this, please!

**Comment by JannikWibker** [comment]
*that's what happens when feedback loops are too long and PR/commit/issue conventions too cumbersome*

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]

File: landing-page/components/sections/ContactSection.tsx [hunk 1]
```diff
@@ -1,17 +1,57 @@
...
+  }
+
   return (
     <TitleSection id="contact" ref={ref} title="Contact">
-      <div className={tw('flex justify-center')}>
-        <GridBox childrenSurroundingClassName='bg-white'>
-            <ContactForm/>
-        </GridBox>
-      </div>
+      <form onSubmit={handleSubmit}>
```
[the remark this answers is not in the document]
Comment (JannikWibker) [reply landing-page/components/sections/ContactSection.tsx:45]: I implemented a basic toast notification on success (and one on error in red)

**Review by MaxSchaefer — APPROVED** [review]
LGTM!
The contact form looks good for now. We should achieve higher UX and GDPR compliance as soon as possible. I'm preparing our HubSpot for that.

**Pull request merged**